splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/FunctionIT.java

```diff
@@ -99,149 +99,164 @@ public void testSinFunction() throws Exception{
+        assertEquals("\n" + sqlText + "\n", expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
+    }
+
+        private void vetThreeArgCoalesce(String sql) throws Exception {
+        // First three values in each row are arguments to COALESCE. The
+                // last value is the expected return value.
+                        String[][] data = {
+                    {"a",  "b",  "c",  "a"},
+                    {null, "b",  "c",  "b"},
+                    {"a",  null, "c",  "a"},
+                    {"a",  "b",  null, "a"},
+                    {null, null, "c",  "c"},
+                    {"a",  null, null, "a"},
+                    {null, "b",  null, "b"},
+                    {null, null, null, null},
+                };
+            PreparedStatement ps = methodWatcher.prepareStatement(sql);
+            for (int i = 0; i < data.length; i++) {
+                ps.setString(1, data[i][0]);
+                ps.setString(2, data[i][1]);
+                ps.setString(3, data[i][2]);
+                ResultSet rs = ps.executeQuery();
+                Assert.assertTrue(rs.next());
+                Assert.assertEquals("Values do not match",rs.getString(1),data[i][3]);
+                Assert.assertFalse(rs.next());
+            }
+    }
 
 	@Test
-	public void testNvlWithDecimalZero() throws Exception {
-		String sqlText = format("select sum(val_no_null)\n" +
-				"from (select nvl(col, 0) as val_no_null from %1$s.B) dt, %1$s.A --splice-properties useSpark=true", FunctionIT.class.getSimpleName());
-
-		String expected = "1   |\n" +
-				"--------\n" +
-				"2.0000 |";
-		ResultSet rs = methodWatcher.executeQuery(sqlText);
-		assertEquals("\n" + sqlText + "\n", expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
-	}
+	public void testCurrentServer() throws Exception {
+        String sqlText = "values current server";
+        String sqlTextAlt = "values current_server";
+        String expected = "1    |\n" +
+                "----------\n" +
+                "splicedb |";
 
-		private void vetThreeArgCoalesce(String sql) throws Exception {
-		// First three values in each row are arguments to COALESCE. The
-				// last value is the expected return value.
-						String[][] data = {
-					{"a",  "b",  "c",  "a"},
-					{null, "b",  "c",  "b"},
-					{"a",  null, "c",  "a"},
-					{"a",  "b",  null, "a"},
-					{null, null, "c",  "c"},
-					{"a",  null, null, "a"},
-					{null, "b",  null, "b"},
-					{null, null, null, null},
-				};
-			PreparedStatement ps = methodWatcher.prepareStatement(sql);
-			for (int i = 0; i < data.length; i++) {
-				ps.setString(1, data[i][0]);
-				ps.setString(2, data[i][1]);
-				ps.setString(3, data[i][2]);
-				ResultSet rs = ps.executeQuery();
-				Assert.assertTrue(rs.next());
-				Assert.assertEquals("Values do not match",rs.getString(1),data[i][3]);
-				Assert.assertFalse(rs.next());
-			}
-	}
+        ResultSet rs = methodWatcher.executeQuery(sqlText);
```

Comment:
You can use "try-with-resources" to ensure resultsets get closed:
try(ResultSet rs = methodWatcher.executeQuery(sqlText)) {
assertEquals("\n" + sqlText + "\n", expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
}

Reply:
Done